test/test_kms_with_cli.py

```diff
@@ -22,16 +22,16 @@ def get_appid_apikey(base_url):
 
 def test_export_datakey(base_url, headers):
     print('====================test_export_datakey start===========================')
-    key_GCM_128 = createkey.createkey(base_url, "EH_AES_GCM_128", "EH_INTERNAL_KEY", None, None, None)
-    key_GCM_192 = createkey.createkey(base_url, "EH_AES_GCM_192", "EH_INTERNAL_KEY", None, None, None)
-    key_GCM_256 = createkey.createkey(base_url, "EH_AES_GCM_256", "EH_INTERNAL_KEY", None, None, None)
-    key_SM4_ctr = createkey.createkey(base_url, "EH_SM4_CTR", "EH_INTERNAL_KEY", None, None, None)
-    key_SM4_cbc = createkey.createkey(base_url, "EH_SM4_CBC", "EH_INTERNAL_KEY", None, None, None)
-
-    key_RSA_3072 = createkey.createkey(base_url, "EH_RSA_3072", "EH_INTERNAL_KEY", None, "EH_PAD_RSA_PKCS1_OAEP", None)
-    key_RSA_4096 = createkey.createkey(base_url, "EH_RSA_4096", "EH_INTERNAL_KEY", None, "EH_PAD_RSA_PKCS1_OAEP", None)
-    key_RSA_2048 = createkey.createkey(base_url, "EH_RSA_2048", "EH_INTERNAL_KEY", None, "EH_PAD_RSA_PKCS1", None)
-    key_SM2 = createkey.createkey(base_url, "EH_SM2", "EH_INTERNAL_KEY", None, None, None)
+    key_GCM_128 = createkey.createkey(base_url, "EH_AES_GCM_128", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_GCM_192 = createkey.createkey(base_url, "EH_AES_GCM_192", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_GCM_256 = createkey.createkey(base_url, "EH_AES_GCM_256", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_SM4_ctr = createkey.createkey(base_url, "EH_SM4_CTR", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_SM4_cbc = createkey.createkey(base_url, "EH_SM4_CBC", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+
+    key_RSA_3072 = createkey.createkey(base_url, "EH_RSA_3072", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_RSA_4096 = createkey.createkey(base_url, "EH_RSA_4096", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_RSA_2048 = createkey.createkey(base_url, "EH_RSA_2048", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
+    key_SM2 = createkey.createkey(base_url, "EH_SM2", "EH_INTERNAL_KEY", "EH_KEYUSAGE_ENCRYPT_DECRYPT")
     symmetricKey = [key_GCM_128, key_GCM_192, key_GCM_256, key_SM4_ctr, key_SM4_cbc]
     asymmetricKey = [key_RSA_3072, key_RSA_4096, key_RSA_2048, key_SM2]
```

Comment:
seems the cases in python is a little ugly, too many duplicate codes, we need to refine them later